src/js/components/SelectMultiple/index.d.ts

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { BasicSelectProps } from '../Select';
+
+export interface SelectMultipleProps extends BasicSelectProps {
+  defaultValue?: (string | number | object)[];
+  help?: React.ReactNode;
```

Comment:
Can this just be a string of text?

Reply:
Yes, added `string` to the type definition